Wikipedia/Localizations/sv.lproj/Localizable.strings

```diff
@@ -325,7 +325,7 @@
 "explore-feed-preferences-global-cards-subtitle" = "Inte språkspecifikt";
 "explore-feed-preferences-global-cards-title" = "Globala kort";
 "explore-feed-preferences-hide-card-action-title" = "Dölj detta kort";
-"explore-feed-preferences-hide-feed-cards-action-title" = "Dölj alla kort av typen “$1”";
+"explore-feed-preferences-hide-feed-cards-action-title" = "Dölj alla \"$1\"-kort";
```

Comment:
These generally look nicer as directional quotes but I'll defer to the editor here